<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Basils-MBP" failures="1" tests="10" name="com.obsqura.assignments.CheckdaysMain" time="0.057" errors="0" timestamp="2022-01-09T17:42:38 NZDT" skipped="0">
  <testcase classname="com.obsqura.assignments.CheckdaysMain" name="testcase2" time="0.002"/>
  <system-out/>
  <testcase classname="com.obsqura.assignments.CheckdaysMain" name="testcase1" time="0.013"/>
  <system-out/>
  <testcase classname="com.obsqura.assignments.CheckdaysMain" name="testcase5" time="0.004"/>
  <system-out/>
  <testcase classname="com.obsqura.assignments.CheckdaysMain" name="testcase7" time="0.002"/>
  <system-out/>
  <testcase classname="com.obsqura.assignments.CheckdaysMain" name="testcase3" time="0.006"/>
  <system-out/>
  <testcase classname="com.obsqura.assignments.CheckdaysMain" name="testcase8" time="0.007"/>
  <system-out/>
  <testcase classname="com.obsqura.assignments.CheckdaysMain" name="testcase9" time="0.003"/>
  <system-out/>
  <testcase classname="com.obsqura.assignments.CheckdaysMain" name="testcase4" time="0.011">
    <failure message="Testcase#4 expected [Monday2] but found [Monday]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Testcase#4 expected [Monday2] but found [Monday]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at com.obsqura.assignments.CheckdaysMain.testcase4(CheckdaysMain.java:69)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- testcase4 -->
  <system-out/>
  <testcase classname="com.obsqura.assignments.CheckdaysMain" name="testcase10" time="0.008"/>
  <system-out/>
  <testcase classname="com.obsqura.assignments.CheckdaysMain" name="testcase6" time="0.001"/>
  <system-out/>
</testsuite> <!-- com.obsqura.assignments.CheckdaysMain -->
